### ISSUE TITLE
Would like to change the background color of the html report

### ISSUE THREAD

**Issue by brennantaylor** [linkedin/LayoutTest-iOS#36]
Alot of our views are white background. White on white is hard to see and twiddling the view or the html page in chrome every time isn't so fun.

Could be nice to either easily change the background of the image being rendered into html or the css on the page or w/e from the test suite, or a dropdown on the page itself.

Just throwing ideas out there.

**Comment by drumnkyle** [comment]
I agree. Should be an easy change. I've been just changing it on the page itself using developer tools in Safari, because I didn't have access to this repo. I'll try to look into changing that soon. Thanks!
